gpAux/gpperfmon/src/gpmon/gpmon_agg.c

```diff
@@ -505,11 +466,11 @@ apr_status_t agg_dup(agg_t** retagg, agg_t* oldagg, apr_pool_t* parent_pool, apr
 		apr_int32_t age = newagg->generation - dp->last_updated_generation - 1;
 		if (age > 0)
 		{
-			if (  (status != GPMON_QLOG_STATUS_SUBMIT
-			       && status != GPMON_QLOG_STATUS_CANCELING
-			       && status != GPMON_QLOG_STATUS_START)
-			   || ((age % 5 == 0) /* don't call is_query_not_active every time because it's expensive */
-			       && is_query_not_active(dp->qlog.key.tmid, dp->qlog.key.ssid, dp->qlog.key.ccnt, active_query_tab, newagg->pool)))
+			if (status == GPMON_QLOG_STATUS_DONE ||
```

Comment:
If I understood correctly, the condition part with status has been inverted. Why GPMON_QLOG_STATUS_SILENT is not added?

Reply:
it is not used anywhere